api/v1/controllers/UploadController.js

```diff
@@ -47,9 +47,7 @@ function _makeFileParams (req, type) {
 function createResumeUpload (req, res, next) {
 	var upload;
 
-	// NOTE: the user object will soon be present on the actual auth object instead
-	// see issue #10
-	var uploadOwner = User.forge({ id: parseInt(req.auth.sub) });
+	var uploadOwner = req.user;
```

Comment:
Nothing has changed for the external functionality/usage of the upload API, so I didn't change the docs. There were changes to the auth docs that I made, though.